tap_aggregator/src/server.rs

```diff
@@ -437,14 +445,14 @@ mod tests {
         allocation_ids: Vec<Address>,
         #[values("0.0")] api_version: &str,
     ) {
-        // Set the request size limit to 10 kB to easily trigger the HTTP 413 error.
-        let small_request_size_limit = 10 * 1024;
+        // Set the request size limit to 100 kB to reproducibly trigger the HTTP 413 error.
+        let http_request_size_limit_100kb = 100 * 1024;
```

Comment:
Should we consider the name `http_request_size_limit`? If `100kB` changes in the future and the name doesn't get updated it could lead to some confusion. If code clarity is the goal maybe do something like:

```rust
        const KILOBYTE: usize = 1024;
        let http_request_size_limit = 100 * KILOBYTE;
```